src/util/counter.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2022 The Yorkie Authors. All rights reserved.
```

Comment:
```suggestion
 * Copyright 2023 The Yorkie Authors. All rights reserved.
```

Reply:
How about renaming file `counter.ts` with `numbers.ts`?

Reply:
Okay, I'll fix it